app-frontend/src/app/components/navBar/navBar.controller.js

```diff
@@ -30,23 +34,4 @@ export default class NavBarController {
     logout() {
         this.authService.logout();
     }
-
-    selectProjectModal() {
```

Comment:
We don't really have a concept of an "active" project anymore - the state is all visible in the URL, so navigation to the project list page makes more sense.  Before, we could leave the project and still have it "active", so having a mechanism for changing the active project was necessary. Now it isn't.